model/iamconfig.go

```diff
@@ -49,8 +49,8 @@ func (c IAMConfig) Validate() error {
 		return errors.New("failed to parse `iam` config: either you set `role.*` options or `instanceProfile.arn` ones but not both")
 	}
 
-	managedPolicyRegexp := regexp.MustCompile(`arn:aws:iam::((\d{12})|aws):policy/([a-zA-Z0-9-=,\\.@_]{1,128})`)
-	instanceProfileRegexp := regexp.MustCompile(`arn:aws:iam::(\d{12}):instance-profile/([a-zA-Z0-9-=,\\.@_]{1,128})`)
+	managedPolicyRegexp := regexp.MustCompile(`arn:aws|aws-us-gov:iam::((\d{12})|aws):policy/([a-zA-Z0-9-=,\\.@_]{1,128})`)
```

Comment:
Probably it should be grouped like `(aws|aws-us-gov)` rather than `aws|aws-us-gov`, so that it won't drop the remaining `:iam::((\d{12})|aws):policy/([a-zA-Z0-9-=,\\.@_]{1,128})` part when the arn started with `arn:aws`?